Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -352,16 +352,20 @@ def audit_formula_name
     }.freeze
 
     def audit_license
+      return unless @strict
```

Comment:
Sorry, I don't think these should all be `@strict`, just the specific case of being deprecated (i.e. the only one that would fail CI currently).

Reply:
Oh, gotcha. That makes sense 👍